tests/dummy/app/components/edit-email.js

```diff
@@ -2,18 +2,18 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   tagName: 'form',
-  session: Ember.computed('email', function() {
-    return this.editEmailService.sessionFor(this.get('email'));
+  state: Ember.computed('email', function() {
```

Comment:
Here we should explicitly call:

```js
editEmailService: Ember.inject.service('edit-email')
```

As I believe this gives better context to how editEmailService is being set